pykickstart/commands/reqpart.py

```diff
@@ -19,14 +19,12 @@
 #
 from pykickstart.version import F23
 from pykickstart.base import KickstartCommand
-from pykickstart.errors import KickstartParseError
 from pykickstart.options import KSOptionParser
 
-from pykickstart.i18n import _
```

Comment:
This needs to remain :)

Reply:
Actually, no, the later commit that adds back the message needs to be changed. Maybe these were applied in the wrong order?

Reply:
Looking at #434 where this should be ported from, I'm similarly confused - the commit order seems to be reversed, and some commits missing.